### PR TITLE
Added Table of Content to README

### LINKED ISSUE #64: Add Table of Contents to README
A quick search reveals this may be a good place to start: https://coderwall.com/p/fy05hq/how-to-make-beautiful-github-readme#team-members

### PR DESCRIPTION
**Pull request by SimonJinaphant**
- Resolve #64

### CODE REVIEW

**Comment by SimonJinaphant** [comment]
@ValRat 
So change `catkin_make run_tests` to `catkin run_tests`
and `catkin_make runtests_MY_PACKAGE_NAME` to `catkin build runtests_MY_PACKAGE_NAME`?

Also noticed that in the in the setup instruction we still have `catkin_make` on step 8. Should that be changed to `catkin build` as well?

**Comment by ValRat** [comment]
Yep, and change step 8 as well. It won't build for a lot of people using `catkin_make` since it'll try to compile the `zed-ros-wrapper` package alongside.

**Comment by SimonJinaphant** [comment]
@ValRat: Changes applied

File: README.md [hunk 1]
```diff
@@ -146,29 +168,25 @@ some_ros_package
 ## Testing
 ### GTest
 - GTest is our primary testing tool at the moment. The ROS wiki has a quick intro to it [here](http://wiki.ros.org/gtest), and we also strongly recommend you read Google's introduction to it [here] (https://github.com/google/googletest/blob/master/googletest/docs/Primer.md), then setup and write a few example tests before you start using it with ROS.
-- Once you've setup your tests in ROS, run `catkin_make run_tests` to run them
-- To run the tests for a specific package, run `catkin_make run_tests_MY_PACKAGE_NAME`
+- Once you've setup your tests in ROS, run `catkin run_tests` to run them
+- To run the tests for a specific package, run `catkin build run_tests_MY_PACKAGE_NAME`
```
Comment (ValRat) [inline README.md]: Last one promise ;)
proper syntax for testing a specific package is :
`catkin build [package_name] --no-deps --catkin-make-args run_tests`
[Source](http://catkin-tools.readthedocs.io/en/latest/verbs/catkin_build.html#building-and-running-tests)

**Comment by ValRat** [comment]
🔥  💯 🔥

**Comment by garethellis0** [comment]
I agree, sexy ToC. Also, cute cat.

**Pull request merged**